core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -160,7 +160,7 @@ protected[controller] class RestAPIVersion(apipath: String, apiversion: String)(
             sendCorsHeaders {
                 (pathEndOrSingleSlash & get) {
                     complete(OK, info)
-                } ~ authenticate(basicauth) {
+                } ~ (authenticate(basicauth) | authenticate(certificateAuth)) {
```

Comment:
First it will use `auth key` to verify, then will failed, then use `client certificate` to verify, so `special any namespace` can't be accessed by `client certificate auth`, because it will use `namespace + subject` from Header to get a Identiy.